client/tx/cli/sign.go

```diff
@@ -44,7 +44,7 @@ recommended to set such parameters manually.
 The --multisig=<multisig_key> flag generates a signature on behalf of a multisig account
 key. It implies --signature-only. Full multisig signed transactions may eventually
 be generated via the 'multisign' command.`,
-		Example: "iriscli tx sign <file> --name <key name> --chain-id=<chain-id>",
+		Example: "iriscli tx sign <file> --name <key-name> --chain-id=<chain-id>",
```

Comment:
A few lines above this line.
`Use:   "sign <file>"` => `Use:   "sign [file]"`

Reply:
only use `[]` in `Use`.

Reply:
I didn't fix the tx and keys related commands because the cosmos-sdk don't modify it, let's decide it in v1.0.0